web/src/hashchange.js

```diff
@@ -436,6 +462,9 @@ function do_hashchange_overlay(old_hash) {
     if (base === "organization") {
         settings.build_page();
         admin.build_page();
+        if (section === "user-list-admin" && settings_panel_menu.org_settings !== undefined) {
+            settings_panel_menu.org_settings.set_user_settings_tab(get_user_settings_tab(section));
+        }
```

Comment:
This probably deserves a brief comment, like:

> The user settings panel has 3 sub-panels with different URLs; mark the correct one to open.

Reply:
I feel like this is definitely a wart of the API that this PR defines; like why can't `admin.launch`, which is passed the full section including the subpanel, be the thing that's responsible for making sure we load the desired subpanel correctly?

To me, it feels like an abstraction violation that `hashchange.js` needs to be doing this setup.

Reply:
Fixed !